### ISSUE TITLE
AWS Lambda and Cognito error on deploy: Only one request to update this UserPool can be processed at a time

### ISSUE THREAD

**Issue by mregnauld** [serverless/serverless#6524]
# This is a Bug Report

## Description

- What went wrong?
  When I deploy lambdas call by Cognito triggers, I got an error.
- What did you expect should have happened?
  No error, and I should see the triggers set correctly in my Cognito user pool.
- What was the config you used?
  Here is my code in the serverless.yml file:
```
functions:

  auto-move-to-user-group:
    handler: src/auto-move-to-user-group.handler
    name: auto-move-to-user-group
    events:
      - cognitoUserPool:
          pool: test-user-pool
          trigger: PostAuthentication
          existing: True

  auto-validation-user-email-modification:
    handler: src/auto-validation-user-email-modification.handler
    name: auto-validation-user-email-modification
    events:
      - cognitoUserPool:
          pool: test-user-pool
          trigger: CustomMessage
          existing: True

resources:

  Resources:

    MyCognitoUserPool:
      Type: AWS::Cognito::UserPool
      Properties:
        UserPoolName: test-user-pool
        AutoVerifiedAttributes:
          - email
        UsernameAttributes:
          - email
        Schema:
          - Name: email
            Required: True

```

- What stacktrace or error message from your provider did you see?
```
An error occurred: AutoDashvalidationDashuserDashemailDashmodificationCustomCognitoUserPool1 - Failed to create resource.
Only one request to update this UserPool can be processed at a time.
```

Similar or dependent issues:

- #4588

## Additional Data

- **_Serverless Framework Version you're using_**:
  1.49.0 (Enterprise Plugin: 1.3.8, Platform SDK: 2.1.0)
- **_Operating System_**:
  Linux Ubuntu 18.04.2 LTS 64 bits
- **_Stack Trace_**:
- **_Provider Error messages_**:
```
An error occurred: AutoDashvalidationDashuserDashemailDashmodificationCustomCognitoUserPool1 - Failed to create resource.
Only one request to update this UserPool can be processed at a time.
```

**Comment by medikoo** [comment]
Thanks for report. It appears as duplicate of #6420, which has been fixed with #6491 (to be published on Wednesday)

**Comment by mregnauld** [comment]
Great news! Thanks!
